src/SIL.XForge.Scripture/ClientApp/src/app/translate/editor/suggestions-settings-dialog.component.html

```diff
@@ -19,7 +19,13 @@
             [(ngModel)]="numSuggestions"
             placeholder="{{ t('number_of_suggestions') }}"
           >
-            <option>1</option> <option>2</option> <option>3</option> <option>4</option> <option>5</option>
+            <mdc-menu>
```

Comment:
Seems odd to put a menu in a select. Ok if works well.

Reply:
Done

Reply:
I think this was addressed in our discussion; that's just the way it's done now (no longer using native select).